bee-message/fuzz/fuzz_targets/fuzz_message.rs

```diff
@@ -3,7 +3,7 @@
 
 #![no_main]
 
-use bee_common::packable::Packable;
+use bee_common::packable::Packable as OldPackable;
```

Comment:
Is this even compiling ?

Reply:
hmm, no... Not even CI catches it does it?

Reply:
Yeah because it's not in the main workspace